app/src/marketplace/modules/myPurchaseList/selectors.js

```diff
@@ -19,34 +24,40 @@ export const selectFetchingMyPurchaseList: (state: StoreState) => boolean = crea
     (subState: MyPurchaseListState): boolean => subState.fetching,
 )
 
+export const selectMyPurchaseListSubscriptionIds: (state: StoreState) => ProductSubscriptionIdList = createSelector(
```

Comment:
`MyPurchaseList` -> `MySubscriptions` etc ?
My vote would be to remove mentions of 'purchase' in favour of 'subscription'